server/issuers.go

```diff
@@ -93,7 +105,13 @@ func (c *Server) issuerGetHandlerV1(w http.ResponseWriter, r *http.Request) *han
 		if !issuer.ExpiresAt.IsZero() {
 			expiresAt = issuer.ExpiresAt.Format(time.RFC3339)
 		}
-		err := json.NewEncoder(w).Encode(issuerResponse{issuer.ID, issuer.IssuerType, issuer.SigningKey.PublicKey(), expiresAt, issuer.IssuerCohort})
+
+		var publicKey *challengebypassristrettoffi.PublicKey
```

Comment:
could this use the already imported alias crypto instead of challengebypassristrettoffi

Reply:
resolved: https://github.com/brave-intl/challenge-bypass-server/pull/145/commits/5c4141f8d098314c19d2434a535c5cd8870f6ba4